tools/ci_build/github/azure-pipelines/templates/windows-build-tools-setup-steps.yml

```diff
@@ -19,18 +19,20 @@ steps:
     #     downloadDirectory: '$(Build.BinariesDirectory)\python'
 
     # Temporary bypass of artifacts permission issue
-    - task: CmdLine@1
-      displayName: 'Download azcopy'
+    - task: PowerShell@2
+      displayName: 'Download AzCopy (used for download test data script)'
       inputs:
-        filename: 'AzCopy.exe'
-        arguments: '/Y /Source:https://onnxruntimetestdata.blob.core.windows.net/models/azcopy.exe /Dest:$(Build.BinariesDirectory)\azcopy.exe'
-        
-    - task: CmdLine@1
-      displayName: 'Download python'
+        targetType: 'inline'
+        script: |
+          Invoke-WebRequest -OutFile $(Build.BinariesDirectory)\azcopy.exe https://onnxruntimetestdata.blob.core.windows.net/models/azcopy.exe
+
+    - task: PowerShell@2
+      displayName: 'Download Python'
       inputs:
-        filename: 'AzCopy.exe'
-        arguments: '/Y /Source:https://onnxruntimetestdata.blob.core.windows.net/models/Miniconda3-4.7.10-Windows-x86_64.exe /Dest:$(Build.BinariesDirectory)\Miniconda3-4.7.10-Windows-x86_64.exe'               
-  
+        targetType: 'inline'
+        script: |
+          Invoke-WebRequest -OutFile $(Build.BinariesDirectory)\Miniconda3-4.7.10-Windows-x86_64.exe https://onnxruntimetestdata.blob.core.windows.net/models/Miniconda3-4.7.10-Windows-x86_64.exe
+
```

Comment:
Powershell Invoke-Webrequest is not the most stable (several builds have random download errors). For larger files, why not use AzCopy instead (since it's already downloaded in step above)? AzCopy is stable, and much faster.

Reply:
I could not get it working

Reply:
What's wrong with the previous one ?

Reply:
The AzCopy preinstalled in the VM cannot seem to access the blob-store (shows resource not available error).

Reply:
The preinstalled AzCopy is quite old (v6.2). The newer version which gets downloaded using powershell is v10.2, and works on the VM.

```
D:\tmp>azcopy copy https://onnxruntimeinstaller.blob.core.windows.net/conda-installer/installer.exe installer.exe
INFO: Scanning...

Job fd0392ca-7ded-f749-49b4-3f03b9ec1a6b has started
Log file is located at: C:\Users\jclient/.azcopy/fd0392ca-7ded-f749-49b4-3f03b9ec1a6b.log

0 Done, 0 Failed, 1 Pending, 0 Skipped, 1 Total,


Job fd0392ca-7ded-f749-49b4-3f03b9ec1a6b summary
Elapsed Time (Minutes): 0.0334
Total Number Of Transfers: 1
Number of Transfers Completed: 1
Number of Transfers Failed: 0
Number of Transfers Skipped: 0
TotalBytesTransferred: 55398376
Final Job Status: Completed
```

Reply:
V10 and the older one, they have totally different syntax.